test/rooms_test.js

```diff
@@ -8,6 +8,7 @@ var expect = require("chai").expect;
 var addHawk = require("superagent-hawk");
 var supertest = addHawk(require("supertest"));
 var sinon = require("sinon");
+var assert = sinon.assert;
```

Comment:
Not sure this is a good idea as this could be easily mistaken for nodejs `assert` module.